core/templates/dev/head/pages/exploration_player/supplemental_card_directive.html

```diff
@@ -64,6 +64,8 @@
   }
 
   .conversation-skin-help-card-content {
+    max-height: 240px;
```

Comment:
![image](https://user-images.githubusercontent.com/5758256/51818377-893c4780-2283-11e9-90ac-8de575e23988.png)

Decrease it by 5 px and I think we'll be good

Reply:
Done